### PR TITLE
docs: add `Quick Start` about running Axon and Blockscan Locally with Docker

### PR DESCRIPTION
**Pull request by Flouse**
## What this PR does / why we need it?

This PR is prepared for the `Quick Start: Running Axon and Blockscan Locally with Docker` documentation.

### What is the impact of this PR?

- No Breaking Change

**Special notes for your reviewer**:
- [x] Shall we delete the [docker-compose](./docker-compose.yml) in file project root?
  Is it being used somewhere?

<details><summary>CI Settings</summary><br/>

### **CI Usage**

**Tip**: Check the CI you want to run below, and then comment `/run-ci`.

**CI Switch**

- [ ] Coverage Test
- [ ] E2E Tests
- [ ] Code Format
- [ ] Web3 Compatible Tests
- [ ] OCT 1-5 And 12-15
- [ ] OCT 6-10
- [ ] OCT 11
- [ ] OCT 16-19
- [ ] v3 Core Tests

### **CI Description**

| CI Name                                   | Description                                                               |
| ----------------------------------------- | ------------------------------------------------------------------------- |
| *Chaos CI*                                | Test the liveness and robustness of Axon under terrible network condition |
| *Cargo Clippy*                            | Run `cargo clippy --all --all-targets --all-features`                     |
| *Coverage Test*                           | Get the unit test coverage report                                         |
| *E2E Test*                                | Run end-to-end test to check interfaces                                   |
| *Code Format*                             | Run `cargo +nightly fmt --all -- --check` and `cargo sort -gwc`           |
| *Web3 Compatible Test*                    | Test the Web3 compatibility of Axon                                       |
| *v3 Core Test*                            | Run the compatibility tests provided by Uniswap V3                        |
| *OCT 1-5 \| 6-10 \| 11 \| 12-15 \| 16-19* | Run the compatibility tests provided by OpenZeppelin                      |

</details>

### CODE REVIEW

**Comment by Flouse** [comment]
> **Special notes for your reviewer**:
> 
> * [x]  Shall we delete the [docker-compose](./docker-compose.yml) file in project root?
>   Is it being used somewhere?

cc @KaoImin

**Review by KaoImin — DISMISSED** [review]

File: devtools/chain/README.md [hunk 1]
```diff
@@ -1,8 +1,77 @@
...
- 7. `nonce = 6` means: Transaction of wckb grants role to crosschain
+# Quick Start: Running Axon and Blockscan Locally with Docker
+
+Run Axon and it's explorer ([blockscan](https://github.com/Magickbase/blockscan)) locally in Docker containers using [docker-compose](https://github.com/docker/compose). Please read [the `docker-compose` file](./docker-compose.yml) to learn more about the services:
+
+- Axon node
+- Blockscan as the explorer of Axon
+- Postgres database
+
+**Note**: This integration is not production ready.
```
Comment (linnnsss) [inline devtools/chain/README.md]: **Note**: This integration is not ready for production use.

File: devtools/chain/README.md [hunk 2]
```diff
@@ -1,8 +1,77 @@
...
+
+```bash
+git clone --depth=1 https://github.com/axonweb3/axon.git
+cd axon/devtools/chain
+docker-compose up -d
+
+
+```
+
+### Watch the logs
```
Comment (linnnsss) [inline devtools/chain/README.md]: ### Watch the Logs

File: devtools/chain/README.md [hunk 3]
```diff
@@ -1,8 +1,77 @@
...
+    "jsonrpc": "2.0",
+    "method": "eth_chainId",
+    "params": []
+}'
+
+# Result: a JSON response containing the chain ID of the Axon devnet.
+{"jsonrpc":"2.0","result":"0x7e6","id":42}
+```
+
+### Accessing [Blockscan](http://127.0.0.1:4020)
```
Comment (linnnsss) [inline devtools/chain/README.md]: ### Access [Blockscan]()

File: devtools/chain/README.md [hunk 4]
```diff
@@ -1,8 +1,77 @@
...
+To access the Blockscan explorer, open your browser and visit http://127.0.0.1:4020. This will allow you to explore the Axon blockchain using the Blockscan interface.
+![Blockscan](blockscan-screenshot.png)
+
+## You are good to go!
```
Comment (linnnsss) [inline devtools/chain/README.md]: ## You Are Good to Go!

**Review by linnnsss — COMMENTED** [review]
Please consider implementing a few subtle modifications to enhance both the style and language

File: devtools/chain/README.md (hunk 1 above)
Comment (Flouse) [reply devtools/chain/README.md]: https://github.com/axonweb3/axon/pull/1364/commits/03063698251a926b32febce5c85e50632bb64ed3

**Review by linnnsss — COMMENTED** [review]
lgtm

**Pull request merged**